pom.xml

```diff
@@ -41,4 +41,28 @@
     <url>http://github.com/jenkinsci/git-plugin</url>
     <tag>HEAD</tag>
   </scm>
+    <!-- Workflow plugin imports are used to test compatibility -->
+  <dependencies>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-job</artifactId>
+      <version>${workflow.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-cps</artifactId>
+      <version>${workflow.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-basic-steps</artifactId>
+      <version>${workflow.version}</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
```

Comment:
@tfennelly This is a good point, went ahead and made this change since it's cleaner and shorter.